### ISSUE TITLE
The Alerts accessibility documentation has a typo - "genrated" and also describes the behavior of aria-live without mentioning it

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4560]
[Clarity Version]
3.1.1

[Test URL]
https://clarity.design/documentation/alerts

[WCAG 2.1 Guideline]
4.1.3 Status Messages
(Level AA)

[OS]
any

[User Agents/Assistive Technologies]
any

[Component]
Alerts

[Description]
The Alerts accessibility documentation has a typo - "genrated" and also describes the behavior of aria-live without mentioning it.

"Accessibility
Actionable controls inside dynamically genrated alerts are not accessible to screen reader users! For this reason Clarity does not recommend using dropdowns, buttons, links inside alerts that appear as dynamic notifications.
Accessibility problems related to using actionable controls inside dynamically generated alerts:

They are announced as part of the alert message, which is out of context and may be confusing. For example, the following alert will be announced as "success acknowledge":
Success
Acknowledge
There is no way for the user to directly interact with the announced action controls."

[User Impact]
Clarity consumers may not understand how to use aria-live with Alert components

[Steps to Reproduce]
- Go to the Test URL 
- Read the accessibility documentation and find the typo and lack of reference to aria-live.

[Recommendation]
1) Fix "genrated"
2) Mention how dynamic textual updates would normally be implemented with aria-live. Controls should not be used inside elements that are using aria-live or one of the related specialty roles like role="alert"
